src/CMakeLists.txt

```diff
@@ -1,62 +1,75 @@
-include(MacroQbtCompilerSettings)
-qbt_set_compiler_options()
-
-include(QbtTargetSources)
-
-find_package(Boost ${requiredBoostVersion} REQUIRED)
-find_package(LibtorrentRasterbar ${requiredLibtorrentVersion} REQUIRED)
-find_package(OpenSSL ${requiredOpensslVersion} REQUIRED)
-
-if (Boost_VERSION_STRING VERSION_LESS 1.60.0)
-    add_definitions(-DBOOST_NO_CXX11_RVALUE_REFERENCES)
+find_package(LibtorrentRasterbar QUIET ${requiredLibtorrentVersion} COMPONENTS torrent-rasterbar)
+if (UNIX AND (NOT APPLE) AND (NOT CYGWIN))
+    if (NOT LibtorrentRasterbar_FOUND)
+        include(FindPkgConfig)
+        pkg_check_modules(LIBTORRENT_RASTERBAR IMPORTED_TARGET GLOBAL "libtorrent-rasterbar>=${requiredLibtorrentVersion}")
+        if (NOT LIBTORRENT_RASTERBAR_FOUND)
+            message(
+                FATAL_ERROR
+                "Package LibtorrentRasterbar >= ${requiredLibtorrentVersion} not found"
+                " with CMake or pkg-config.\n- Set LibtorrentRasterbar_DIR to a directory containing"
+                " a LibtorrentRasterbarConfig.cmake file or add the installation prefix of LibtorrentRasterbar"
+                " to CMAKE_PREFIX_PATH.\n- Alternatively, make sure there is a valid libtorrent-rasterbar.pc"
+                " file in your system's pkg-config search paths (use the system environment variable PKG_CONFIG_PATH"
+                " to specify additional search paths if needed)."
+            )
+        endif()
+        add_library(LibtorrentRasterbar::torrent-rasterbar ALIAS PkgConfig::LIBTORRENT_RASTERBAR)
+        # force a fake package to show up in the feature summary
+        set_property(GLOBAL APPEND PROPERTY
+            PACKAGES_FOUND
+            "LibtorrentRasterbar via pkg-config (required version >= ${requiredLibtorrentVersion})"
+        )
+        set_package_properties("LibtorrentRasterbar via pkg-config (required version >= ${requiredLibtorrentVersion})"
+            PROPERTIES
+            TYPE REQUIRED
+        )
+    else()
+        set_package_properties(LibtorrentRasterbar PROPERTIES TYPE REQUIRED)
```

Comment:
@glassez 

If the package is found without pkg-config, we set the `TYPE` to `REQUIRED`, so that it displays correctly as a required package. This is all due to the fact that on Linux, we first find libtorrent as a non-`REQUIRED` package to give it a second chance with pkg-config.